src/solvers/flattening/boolbv.cpp

```diff
@@ -201,7 +201,7 @@ bvt boolbvt::convert_bitvector(const exprt &expr)
     if(expr.operands().size()!=1)
       return conversion_failed(expr);
 
-    std::size_t op_width=boolbv_width(expr.op0().type());
+    std::size_t op_width = boolbv_width(to_unary_expr(expr).op().type());
```

Comment:
I'm not sure that width is a unary_exprt...

Reply:
I'm not sure there's anything wrong here?